script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -569,7 +568,7 @@ I don't have a lot of money in my wallet, but I should at least get something li
 [sha:b8ee4d157d06152621d98bf25349091876ccae71]{
 -- Page 40, Offset 16670.
 -- こんな日にかぎって食欲がある事が恨めしい。
-I resent the fact that I actually have an appetite today of all days.
+I resent the fact that I actually have an appetite, today of all days.
```

Comment:
I don't like seperating 'appetite' and 'today' but dont have a *great* alternative. 
"I resent the fact that, of all days, I actually have an appetite today"
I might be over-thinking it and or delusional though.

Reply:
I pretty strongly disagree just from a readability standpoint. I think it's fine as-is